src/status_im/contexts/onboarding/enable_notifications/view.cljs

```diff
@@ -71,11 +71,7 @@
   []
   (let [insets (safe-area/get-insets)]
     [rn/view {:style (style/page-container insets)}
-     [rn/view {:style style/page-heading}
-      [quo/page-nav
```

Comment:
I think we still will be able to go back using hardware back button.
Better to call :init-root, instead of `:navigate-to` while opening enable notifications screen